test/CMakeLists.txt

```diff
@@ -1020,52 +1020,6 @@ add_custom_test(smoke_solver_ConvMlirIgemm_W GFX900_DISABLED GFX908_DISABLED GFX
     ENVIRONMENT MIOPEN_FIND_ENFORCE=SEARCH_DB_UPDATE MIOPEN_DEBUG_TUNING_ITERATIONS_MAX=5 ${IMPLICITGEMM_MLIR_ENV_W}
     COMMAND $<TARGET_FILE:test_conv2d> ${TEST_CONV_VERBOSE_W} --input 64 64  28 28 --weights 64  64  1 1 --pads_strides_dilations 0 0 1 1 1 1 ${MIOPEN_TEST_FLAGS_ARGS}
 )
-
-add_custom_test(test_regression_half_vega SKIP_UNLESS_ALL FLOAT_DISABLED HALF_ENABLED GFX908_DISABLED GFX90A_DISABLED
-# Issue #1956.
-    ENVIRONMENT MIOPEN_FIND_MODE=normal MIOPEN_DEBUG_FIND_ONLY_SOLVER='GemmBwdRest'
-    COMMAND $<TARGET_FILE:test_conv3d> ${TEST_CONV_VERBOSE_B} --cmode conv --pmode default --group-count 1 --batch_size 2 --input_channels 64 --output_channels 32 --spatial_dim_elements 128 128 128 --filter_dims 3 3 3 --pads_strides_dilations 1 1 1 1 1 1 1 1 1 --trans_output_pads 0 0 0 --in_layout NCDHW --fil_layout NCDHW --out_layout NCDHW ${MIOPEN_TEST_FLAGS_ARGS}
-)
-# Test case for issue #1956 uses huge tensors, therefore:
-set_tests_properties(test_regression_half_vega PROPERTIES RUN_SERIAL On)
```

Comment:
Hmm, interesting. That may be hard to support in gtest-paralle infrastructure. @atamazov @JehandadKhan  FYI

Reply:
@CAHEK7 @xinlipn @JehandadKhan Yes. The `RUN_SERIAL` tests should be either put into a different binary or selected with `--gtest-filter`. I suspect that in any case we'll need to run gtest-parallel twice, like this:
```bash
$ .../gtest-parallel miopen_gtest --gtest_filter=<REGEX-TO-EXCLUDE-ALL-SERIAL-TESTS>
$ .../gtest-parallel miopen_gtest --gtest_filter=<REGEX-TO-INCLUDE-ONLY-SERIAL-TESTS> --workers=1
```